docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,17 +3,179 @@ title: Database Access AWS IAM Reference
 description: AWS IAM policies for Teleport database access.
 ---
 
+## Auto-discovery
 Teleport automatically discovers and configures IAM for RDS and Redshift given
 proper IAM permissions as described in the [AWS RDS & Aurora
 guide](../guides/rds.mdx) and the [AWS Redshift
 guide](../guides/postgres-redshift.mdx).
 
+The Teleport Database Service requires a set of AWS IAM permissions to:
+- update database configurations
+- update its identity with connection-related permissions
+
+You can generate and manage the permissions with the [`teleport db configure
+bootstrap`](../../database-access/reference/cli.mdx#teleport-db-configure-bootstrap)
+command. For example, the following command would generate and print the
+IAM policies:
+
+```code
+$ teleport db configure bootstrap --manual
+```
+
+Or if you prefer, you manage the IAM permissions yourself. Examples of policies
+for each discovery type are shown below.
+
+### Aurora/RDS
+
+<Tabs>
+  <TabItem label="RDS / IAM user">
+  Use this policy if you're connecting to RDS instances and your Teleport
+  database agent runs as an IAM user (for example, uses an AWS credentials file).
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "rds:DescribeDBInstances",
+                  "rds:ModifyDBInstance",
+                  "iam:GetUserPolicy",
+                  "iam:PutUserPolicy",
+                  "iam:DeleteUserPolicy"
+              ],
+              "Resource": "*"
```

Comment:
As we've discussed today, let's update this to not require wildcard for *UserPolicy actions here and below.